nullaway/src/test/resources/com/uber/nullaway/testdata/CheckContractPositiveCases.java

```diff
@@ -40,7 +40,33 @@ Object foo(Object a, @Nullable Object b) {
   @Nullable
   Object fooTwo(Object a, @Nullable Object b) {
     if (b != null) {
-      // BUG: Diagnostic contains: Method has @Contract
+      // BUG: Diagnostic contains: Method fooTwo has @Contract(_, !null -> !null), but this appears
+      // to be violated,
+      // as a @Nullable value may be returned when parameter b is non-null.
```

Comment:
I know this was GJF breaking up these lines, but worth formatting them reasonably. For most of this is as easy as deleting the manual space you had added :)

Reply:
Yeah, this looked weird